apps/src/storage/getColumnDropdown.js

```diff
@@ -40,9 +39,8 @@ export function getTables() {
 
 export function getColumns() {
   return function(editor) {
-    const tableName = stripQuotes(
-      getFirstParam('getColumn', this.parent, editor)
-    );
+    const firstParam = getFirstParam('getColumn', this.parent, editor);
+    const tableName = firstParam.substring(1, firstParam.length - 1);
```

Comment:
should we add a unit test that tests this bug so we don't regress?

Reply:
did a little refactor to make this testable, ptal!